locales-pending/premium.ftl

```diff
@@ -253,16 +254,44 @@ dashboard-top-banner-your-data-is-protected-description =
   }
 dashboard-top-banner-your-data-is-protected-cta = See what’s fixed
 
-# Variables: 
+# Variables:
 # $starting_exposure_total_num is the number of exposures the user has resolved.
 dashboard-top-banner-your-data-is-protected-all-fixed-description = 
   { $starting_exposure_total_num ->
-    [one] Great work, you’ve fixed 1 exposure of your data! Upgrade to { -brand-premium } for continuous protection.
-    *[other] Great work, you’ve fixed all { $starting_exposure_total_num } exposures of your data! Upgrade to { -brand-premium } for continuous protection.
+    [one] Great work, { $starting_exposure_total_num } exposure of your data is fixed! Upgrade to { -brand-premium } and wel’ll continue to monitor for new exposures. Plus we’ll automatically remove your info from any sites that are selling it.
+    *[other] Great work, all { $starting_exposure_total_num } exposures of your data are fixed! Upgrade to { -brand-premium } and wel’ll continue to monitor for new exposures. Plus we’ll automatically remove your info from any sites that are selling it.
   }
-dashboard-top-banner-your-data-is-protected-all-fixed-cta = Get Continuous Protection
+dashboard-top-banner-your-data-is-protected-all-fixed-cta = Get continuous protection
+dashboard-top-banner-non-us-no-exposures-found-description = Great news! We searched all known data breaches and found no exposures. We’ll keep monitoring your email address and will alert you if a new breach occurs.
 
-# Variables: 
+# Variables:
+# $exposures_total_num is the total number of exposures the user has.
+# $data_breach_total_num is the total number of data breaches the user has.
+dashboard-top-banner-non-us-protect-your-data-description = {
```

Comment:
Yes! We had a discussion regarding an [almost identical string](https://github.com/mozilla/blurts-server/pull/3431#discussion_r1331775867) and I checked back with content. Ultimately, we decided to stick with that pattern and it was approved.

Maybe we can think of moving the messaging of those two strings closer together so that the first part matches and we only need one string. But that would only help I guess if it’s fine to separate the second part out into its own string ID?

Reply:
If Flod's OK with it I'm OK with it :)